src/modules/quest/models/Quest.js

```diff
@@ -0,0 +1,30 @@
+const knex = require('../../../data/connection')
+
+module.exports = () => {
+  const TABLE_NAME = 'tb_quest'
+
+  const create = async data => await knex.insert(data).into(TABLE_NAME)
+
+  const list = async data => knex.select('*').from(TABLE_NAME)
+
+  const find = async id => {
+    const result = await knex.select('*').from(TABLE_NAME).where({ id }).first().then(row => row)
+    if (!result) { throw new Error('Quest not found') }
```

Comment:
`Questionnaire not found` (ou, se o caso aqui for buscar uma pergunta apenas e não o questionário todo, `Question not found`).